utils/vlog/tests/muxes/routing/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+mux_gen(
+  NAME rmux
+  INPUTS I0,I1
```

Comment:
So if I understand how this works you are implicitly defining the "fasm_mux" metadata value to be defined as the name of the input?  I don't think that is sufficient in the general case, because the FASM tag might include ``.`` (periods).  Maybe default to INPUTS and allow a second explicit FASM tag array?

Also consider the NULL case.

Take this as an example: https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/xc7/primitives/common_slice/common_slice.pb_type.xml#L475

Reply:
@kgugala